src/presets/vercel/utils.ts

```diff
@@ -187,6 +187,25 @@ function generateBuildConfig(nitro: Nitro, o11Routes?: ObservabilityRoute[]) {
           }
           return route;
         }),
+      // Skew protection
+      ...(process.env.VERCEL_SKEW_PROTECTION_ENABLED
+        ? [
+            {
+              src: "/.*",
+              has: [
+                {
+                  type: "header",
+                  key: "Sec-Fetch-Dest",
+                  value: "document",
+                },
+              ],
+              headers: {
+                "Set-Cookie": `__vdpl=${process.env.VERCEL_DEPLOYMENT_ID}; Path=${nitro.options.baseURL}; SameSite=Strict; Secure; HttpOnly`,
```

Comment:
To double check in conjunction with #3792, does it still work on vercel with manually overriden deploymentId?